backend/src/Altinn.AccessManagement.UI/Altinn.AccessManagement.UI/Controllers/LookupController.cs

```diff
@@ -23,20 +23,21 @@ public class LookupController : ControllerBase
         /// <param name="lookupService">service implementation for lookups</param>
         public LookupController(
             ILogger<LookupController> logger,
-            ILookupService lookupService)
+            ILookupService lookupService,
+            IHttpContextAccessor httpContextAccessor)
         {
             _logger = logger;
             _lookupService = lookupService;
+            _httpContextAccessor = httpContextAccessor;
         }
-
+        
         /// <summary>
         /// Endpoint for retrieving delegated rules between parties
         /// </summary>
         /// <response code="400">Bad Request</response>
         /// <response code="500">Internal Server Error</response>
         [HttpGet]
-        [Authorize]
```

Comment:
Implementation still requires the [Authorize] attribute

Reply:
Reverted changes